src/main/java/com/uid2/shared/store/parser/ClientParser.java

```diff
@@ -10,6 +10,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
+// change to return simple list
```

Comment:
What is this referring to?

Reply:
This is a note for you :) 

In this parser, we are building a dictionary from keys to ClientKey, this can be changed to return a list of ClientKey instead

Removed the comment